frontend/src/views/CanteenEditor/CanteenForm.vue

```diff
@@ -342,6 +353,32 @@ export default {
       siretFormIsValid: true,
       duplicateSiretCanteen: null,
       siretDialog: false,
+      ministries: [
+        { value: "premier_ministre", text: "Service du premier ministre" },
```

Comment:
```suggestion
        { value: "premier_ministre", text: "Service du Premier ministre" },
```
https://www.gouvernement.fr/le-premier-ministre

Reply:
Fix : https://github.com/betagouv/ma-cantine/pull/1273/commits/74db159fa3452d7e19c962d9646ffc238875f86f